src/commands/codepush/lib/react-native-utils.ts

```diff
@@ -571,7 +571,11 @@ function getComposeSourceMapsPath(): string {
 
 function getCliPath(): string {
   if (process.platform === "win32") {
```

Comment:
Should  the path be handled on other platforms or is it just a windows issue?

Reply:
Yes, this issue only reproduces on Windows.